### ISSUE TITLE
.screen-reader-text class

### ISSUE THREAD

**Issue by johnpennypacker** [uriweb/uri-modern#106]
The current CSS includes .screen-reader-text in a descendant selector, but that doesn't cover every instance, there should be a style for the class selector by itself.
